code/modules/mob/living/carbon/xenomorph/abilities/shrieker/shrieker_powers.dm

```diff
@@ -0,0 +1,219 @@
+/datum/action/xeno_action/activable/shriek/use_ability(atom/A)
+	var/mob/living/carbon/Xenomorph/xeno = owner
+	if (!istype(xeno))
+		return
+
+	if (!action_cooldown_check())
+		return
+	if (!check_plasma_owner())
+		return
+
+	if(!xeno.check_state())
+		return
+
+	if (xeno.mutation_type == SHRIEKER_NORMAL)
+		var/datum/behavior_delegate/shrieker/BD = xeno.behavior_delegate
+		if (!istype(BD))
+			return
+		if (!BD.use_internal_acid_ability(shriek_cost))
+			return
+
+	if (curr_effect_type == SHRIEKER_SHRIEK_BUFF)
+
+
+		playsound(xeno.loc, shriek_sound_effectt, 55, 0, status = 0)
```

Comment:
the fourth parameter 0 is used as a boolean for TRUE and FALSE - set it to FALSE, not 0, for readability. same for status if it's used as a boolean as well